minio/minio
# Support https and basic-auth for elasticsearch notification target

## Description
Add basic-auth and https support for ES notification target

## Motivation and Context
To support HTTPs for ES notification target
Fixes https://github.com/minio/minio/issues/10262

## How to test this PR?
- Start elastic search with SSL enabled.
- Place the ca.cert in `~/.minio/certs/CAs/`
- Set up bucket notification and check for events
Ref - https://www.elastic.co/blog/configuring-ssl-tls-and-https-to-secure-elasticsearch-kibana-beats-and-logstash

## Types of changes

## Checklist:
